### DISCUSSION TITLE
Is react-flow is free to use for commercial products under MIT license

### DISCUSSION THREAD

**Discussion by harshajayaweeraXHJ** [xyflow/xyflow#3397] [General]
If I'm not using any pro features is react-flow is really free to use in commercial products?
I see following sentence in Readme

_"Are you using React Flow at your organization and making money from it? Awesome! We rely on your support to keep React Flow developed and maintained under an MIT License"_

When you say "rely on support" it does not imply we should purchase it to use in commercial products right?
Unless we are using pro components.

Can you please clarify. If i'm not using pro features if it free for any commercial or personal projects under MIT license?

**Comment by bcakmakoglu** [comment]
[This discussion should help.](https://github.com/wbkd/react-flow/discussions/2961)

**Comment by moklick** [comment]
You don't need a subscription to use React Flow within a commercial product. It's MIT licensed so there is no need to pay anything. There are also no pro features. With a subscription you get access to pro examples and you secure the further development.
